src/Renci.SshNet/ScpClient.NET.cs

```diff
@@ -268,5 +283,11 @@ private void InternalDownload(IChannelSession channel, Stream input, FileSystemI
                 SendConfirmation(channel, 1, string.Format("\"{0}\" is not valid protocol message.", message));
             }
         }
+
+        private static string SanitizeUnixFilenameForWindows(string fileName)
+        {
+            fileName = Path.GetInvalidFileNameChars().Aggregate(fileName, (current, c) => current.Replace(c.ToString(), string.Empty));
```

Comment:
All hail the mighty Linq!

I think that was a resharper replacement, I'm just used to seeing Linq everywhere. Imagine that will go when I add the logging.